### PR TITLE
Lf/home book nav

### PR DESCRIPTION
**Pull request by FrazierLE**
#### What does this PR do?
This one does too much. It holds the Home component, updated Navbar component, Book Details component, and Book component. 

#### Where should your reviewer start
Npm i then start with the Home component and navigate the page to the book details page. Play with the hamburger menu and the search feature. I haven't add any lower case functionality so to test the search bar capitalize the book title. 

#### What (if any) features are you implementing
Search bar on home page to filter the book search,  book details page, navbar routing menu

#### Were there any issues that arose?
Routing and page refresh

#### Is there anything that you need from your teammate?
Extra pairs of eyes for feedback/their thoughts

### CODE REVIEW

File: src/Components/BookDetails/BookDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      users {
+        userName,
+        emailAddress,
+        location
+      }
+    }
+  }
+`
+
+const BookDetails = () => {
```
Comment (Adelle-Pitsas) [inline src/Components/BookDetails/BookDetails.tsx:23]: I was getting an error in my console to add a key to this component -- might just be good to add it here (we can make it the id of the book) so the console doesn't keep yelling at us

File: src/Components/Home/Home.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+const BOOKS_DATA = gql `
+    query books {
+      books {
+      id 
+      imageUrl
+    }
+}`
+
+interface book {
```
Comment (Adelle-Pitsas) [inline src/Components/Home/Home.tsx:24]: We should probably meet to discuss putting these into their own files!

**Comment by Adelle-Pitsas** [comment]
Just something that I've noticed while navigating the app--there are times when I click on `My Account` in the nav bar but I'm logged out. When I inspect it it looks like the word is not inline with where its clickable for the user (see pic below) might just be worth fixing that quick before merging! 
image

File: src/Components/Home/Home.tsx (hunk 2 above)
Comment (FrazierLE) [reply src/Components/Home/Home.tsx:24]: Yeah I think that's a good idea!

**Comment by Adelle-Pitsas** [comment]
Looks great! Thanks for making those changes!

**Pull request merged**